netmesh/model/netmesh/netmesh.pb.go

```diff
@@ -65,38 +53,16 @@ func (m *NetworkServiceEndpoint) GetUuid() string {
 }
 
 type NetworkService struct {
-	Name                 string                           `protobuf:"bytes,1,opt,name=name" json:"name,omitempty"`
-	Uuid                 string                           `protobuf:"bytes,2,opt,name=uuid" json:"uuid,omitempty"`
-	Selector             string                           `protobuf:"bytes,3,opt,name=selector" json:"selector,omitempty"`
-	Channels             []*NetworkService_NetmeshChannel `protobuf:"bytes,4,rep,name=channels" json:"channels,omitempty"`
-	XXX_NoUnkeyedLiteral struct{}                         `json:"-"`
-	XXX_unrecognized     []byte                           `json:"-"`
-	XXX_sizecache        int32                            `json:"-"`
-}
-
-func (m *NetworkService) Reset()         { *m = NetworkService{} }
-func (m *NetworkService) String() string { return proto.CompactTextString(m) }
-func (*NetworkService) ProtoMessage()    {}
-func (*NetworkService) Descriptor() ([]byte, []int) {
-	return fileDescriptor_netmesh_9e195ebaabd0ca3b, []int{1}
-}
-func (m *NetworkService) XXX_Unmarshal(b []byte) error {
-	return xxx_messageInfo_NetworkService.Unmarshal(m, b)
-}
-func (m *NetworkService) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
-	return xxx_messageInfo_NetworkService.Marshal(b, m, deterministic)
-}
-func (dst *NetworkService) XXX_Merge(src proto.Message) {
-	xxx_messageInfo_NetworkService.Merge(dst, src)
-}
-func (m *NetworkService) XXX_Size() int {
-	return xxx_messageInfo_NetworkService.Size(m)
-}
-func (m *NetworkService) XXX_DiscardUnknown() {
-	xxx_messageInfo_NetworkService.DiscardUnknown(m)
+	Name     string                           `protobuf:"bytes,1,opt,name=name" json:"name,omitempty"`
+	Uuid     string                           `protobuf:"bytes,2,opt,name=uuid" json:"uuid,omitempty"`
+	Selector map[string]string                `protobuf:"bytes,3,rep,name=selector" json:"selector,omitempty" protobuf_key:"bytes,1,opt,name=key" protobuf_val:"bytes,2,opt,name=value"`
```

Comment:
Because this changed, you'll want to regenerate the CRD code as well. To do that, run something like the following:

```
GOPATH=<path to go root> ./scripts/update-codegen.sh
```

Reply:
@mestery I have run it already, that is why there were a bunch of `generated` files updated and part of this PR. Just in case I re-run it again and no change was detected by git.
```
SBEZVERK-M-L0L4:networkservicemesh sbezverk$ ./scripts/update-codegen.sh
Calling ./vendor/k8s.io/code-generator/generate-groups.sh
Generating deepcopy funcs
Generating clientset for networkservicemesh.io:v1 at github.com/ligato/networkservicemesh/pkg/client/clientset
Generating listers for networkservicemesh.io:v1 at github.com/ligato/networkservicemesh/pkg/client/listers
Generating informers for networkservicemesh.io:v1 at github.com/ligato/networkservicemesh/pkg/client/informers
Generating other deepcopy funcs
SBEZVERK-M-L0L4:networkservicemesh sbezverk$ git status
On branch selector_map
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	cmd/netmesh/debug

nothing added to commit but untracked files present (use "git add" to track)
```

Reply:
I was reviewing this on my phone and missed those files, my apologies.